### DISCUSSION TITLE
Help with iOS implementation

### DISCUSSION THREAD

**Discussion by Codegn** [JuulLabs/kable#638] [Q&A]
Hello everyone,

We have been successfully using Kable in an Android app, but we are a bit at a loss trying to implement the same features in iOS. I was wondering if any of you have some examples of an iOS implementation or even some guidance on the steps that need to be followed in order to accomplish this.

Best regards!

**Comment by twyatt** [comment]
If for a Kotlin multiplatform Compose app (that is also targeting iOS), then [SensorTag sample](https://github.com/JuulLabs/sensortag) might be a good guide?
